tfx/utils/path_utils.py

```diff
@@ -72,11 +95,32 @@ def serving_model_dir(output_uri: Text) -> Text:
 
 
 def serving_model_path(output_uri: Text) -> Text:
-  """Returns path for timestamped and named serving model exported."""
-  export_dir = os.path.join(serving_model_dir(output_uri), 'export')
+  """Returns path for exported serving model."""
+  model_dir = serving_model_dir(output_uri)
+  export_dir = os.path.join(model_dir, 'export')
   if tf.io.gfile.exists(export_dir):
+    # TODO(b/160795287): Deprecate estimator based executor.
+    absl.logging.warning('Support for estimator-based executor and model export'
+                         ' will be deprecated soon. Please use export structure '
+                         '<ModelExportPath>/serving_model_dir/saved_model.pb"')
     model_dir = io_utils.get_only_uri_in_dir(export_dir)
     return io_utils.get_only_uri_in_dir(model_dir)
   else:
     # If dir doesn't match estimator structure, use serving model root directly.
-    return serving_model_dir(output_uri)
+    return model_dir
+
+
+# TODO(b/160795287): Deprecate estimator based executor.
+def copy_model(working_dir: Text, dest: Text, tag: Text) -> None:
```

Comment:
I feel this function only change 2 lines[1] of code to 1 lines[2], but the tag is a risky (you can pass the function in this function directly) and the readability is not as good as the two line solution (drop function and inline the code), wdyt?

[1]
source = path_fn(working_dir)
  io_utils.copy_dir(source, dest)
[2]
copy_model

Reply:
agreed, would make readability much easier and will remove need for extra if statement. will fix

Reply:
I'm leaning towards keeping the copy_model fn (since it does somewhat improve readability). Additionally, it also links the executor to the path_utils file (which describes the folder structure deprecation in more detail), in case users have difficulty understanding what's needed for the deprecation

Reply:
I mean we can just do [1] in place instead of call this function, it also includes path_utils

another option is [2]

[1]
source = path_utils.serving_model_path(xxx)
io_utils.copy_dir(source, dest)

[2] 
def copy_model(working_dir, dest, model_extract_function):
   ...

then 
copy_model(working_dir, dest, path_utils.serving_model_path) thus you don't need to manually write the tag.

Reply:
gotcha, the only two reasons for the tag was (1) choosing path_fn (which is now just passed in) and (2) for the log message to be accurate. If we just do it in-line both of these issues can be resolved

Reply:
i think in-place might be preferable, will switch it to that